doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc

```diff
@@ -1,9 +1,9 @@
 [id='clustering-datagrid-proc_{context}']
 = Installing and configuring {DATAGRID}
 
-Install and configure {DATAGRID} for the {PRODUCT} clustered environment for more efficient searching across cluster nodes.
+For more efficient searching across cluster nodes, install {DATAGRID} and configure it for the {PRODUCT} clustered environment.
 
-For information about advanced installation and configuration options, and {DATAGRID} modules for {EAP}, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
+For information about advanced installation and configuration options and {DATAGRID} modules for {EAP}, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
```

Comment:
[Style] Does removing the comma here cause "advanced installation and configuration options" to refer to {EAP} rather than to Data Grid?

s/advanced installation and configuration options and {DATAGRID} modules for {EAP}/advanced installation and configuration options for {DATAGRID}, and {DATAGRID} modules for {EAP}.

Reply:
No. The comma there is incorrect. However, on second look the sentence is unclear so rewrote:
For information about Red Hat Data Grid advanced installation and configuration options and Red Hat Data Grid modules for Red Hat JBoss EAP, see the Red Hat Data Grid User Guide.